src/Sylius/Bundle/ApiBundle/Resources/config/services/filters.xml

```diff
@@ -39,6 +39,13 @@
             <tag name="api_platform.filter" />
         </service>
 
+        <service id="sylius.api.shipping_method_order_filter" parent="api_platform.doctrine.orm.order_filter">
+            <argument type="collection">
+                <argument key="code"/>
```

Comment:
Is it working ok, if we don't have position in this collection?

Reply:
Yes 😄 In fact, in our _sorting by position_ scenarios we don't sort shipping methods 💃 We just check does the default sorting by position works well :rock